app/models/manifest.rb

```diff
@@ -29,6 +31,12 @@ def initialize(path, opts)
     @name = opts.fetch("name", default_name)
     @host = opts.fetch("host", default_host)
     @notes = opts.fetch("notes", default_notes)
+
+    if !!opts.fetch("notes_is_markdown", false)
+      renderer = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: true)
+      @notes = renderer.render(@notes)
+    end
```

Comment:
[This already exists](https://github.com/OSC/ood_appkit#markdown-handler): `OodAppkit.markdown`

So you should be able to just do this:

```
@notes = OodAppkit.markdown.render(@notes)
```

In fact [we do that in the Dashboard](https://github.com/OSC/ood-dashboard/blob/4bd24f17cd5fafb98e5cf34a190d1d9e273798d9/app/views/batch_connect/session_contexts/new.html.erb#L54).